services/forms/user_form.go

```diff
@@ -377,6 +377,16 @@ func (f *AddSecretForm) Validate(req *http.Request, errs binding.Errors) binding
 	return middleware.Validate(errs, ctx.Data, f, ctx.Locale)
 }
 
+type EditVariableForm struct {
+	Name string `binding:"Required;MaxSize(50)"`
+	Data string `binding:"Required"`
```

Comment:
Does this and "secret.Data" need a "MaxSize" limitation? Otherwise is it possible for users to store mega or giga bytes into database?

Reply:
Some configurable limit might be good to prevent abuse, yes. I'm thinking maybe 16KB default, but could be done later.

Reply:
The limit of 50 for `Name` does seem a bit low and I could see that being hit with long variable names. How about 255?

Reply:
It does need a limitation. The size of a single varibale or secret is limited to 48KB on [GH](https://docs.github.com/en/actions/learn-github-actions/variables#limits-for-configuration-variables), but the `MaxSize` tag counts the number of characters, how about use `len()` to count bytes?

Reply:
It doesn't need to be that accurate, as long as `MaxSize(65535)` doesn't harm anyone.